.readthedocs.yaml

```diff
@@ -0,0 +1,13 @@
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  commands:
+    - mkdir --parents $READTHEDOCS_OUTPUT/html/
+    - pip install nox
```

Comment:
I noticed that most YAML documents under `.github/` use no-space indents for sequences. Why not follow the same style? Here's a linting config I tend to use: https://github.com/cherrypy/cheroot/blob/main/.yamllint.